### PR TITLE
feat: Add rating component

### PR DESCRIPTION
**Pull request by clucasalcantara**
### Story
This aims to add the rating component using only emotion and hooks.

### Preview
rating-component

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,30 +1,35 @@
 {
```
Comment (mvfsillva) [inline .eslintrc]: What do you think about using XO?

I think this file should be another PR.

File: package.json [hunk 2]
```diff
@@ -27,19 +27,20 @@
     "react": "^16.8.6",
     "react-autosuggest": "^9.4.3",
     "react-dom": "^16.8.6",
+    "react-icons": "^3.5.0",
     "react-instantsearch-dom": "^5.3.2",
     "react-router-config": "^4.4.0-beta.6",
     "react-router-dom": "^4.3.1",
     "react-scripts": "2.1.3",
-    "react-test-renderer": "^16.7.0"
+    "react-test-renderer": "^16.7.0",
```
Comment (mvfsillva) [inline package.json]: Do we need both?
`react-test-renderer` and `enzyme`

File: .eslintrc (hunk 1 above)
Comment (clucasalcantara) [reply .eslintrc]: Yes, I agree with you but, I've made these changes here just to have available asap during the rating development and forgot to remove :P 

Lets keep it here since we gonna send more PR's about structure refactoring :)

File: package.json (hunk 2 above)
Comment (clucasalcantara) [reply package.json]: nice catch, removed test renderer
